src/components/modal/Modal.jsx

```diff
@@ -136,6 +143,24 @@ const HomeHelpTooltip = () => {
   );
 };
 
+const FaviconHelpTooltip = () => {
```

Comment:
I actually think we don't need this. if the user doesn't provide a favicon, the code automatically takes the prefix url and appends /favicon.ico to the root. So this is more for if the user wants to find a higher res favicon that isn't the default /favicon.ico one. You can still have a help tooltip explain this if u think it's necessary